include/crm/common/alerts_internal.h

```diff
@@ -30,6 +30,8 @@ typedef struct {
     int timeout;
     char *tstamp_format;
     char *recipient;
+    char *select_kind;
+    char *select_attribute_name;
```

Comment:
Instead of storing the comma-delimited strings here, and parsing them every time we need to do a comparison, let's store these as a char** and parse it when reading it from the config. That way, the comparisons will be quicker. To store it as a char**, we can either add another field for the number of entries, or make the last entry NULL as a marker.